src/app/loans/loans-routing.module.ts

```diff
@@ -111,10 +112,13 @@ const routes: Routes = [
           }
         },
         {
-          path: ':action',
-          component: LoanAccountActionsComponent,
-          data: { title: extract('Loan Account Actions'), routeParamBreadcrumb: 'action' },
-        },
```

Comment:
@BLasan You don't need to remove the routing for ':action', that's a generic routing to be used for all the action buttons. You can try using the close action and look for its implementation, the other actions have to be implemented in the same way. And as the action route is made generic you don't need to add separate routes for every button.

Also there is a convention for the action buttons, and its like, from the name of the button in the web-app, replacing the spaces with '-' and making it lowercase.You can find the code for the same in the file, src/app/loans/loans-view/loans-view.component.ts, the loanAction().